README.md

```diff
@@ -89,9 +89,13 @@ the Agent and VPP on [Dockerhub][14].
 1. Run VPP + VPP Agent in a Docker image:
 ```
 docker pull ligato/vpp-agent
-docker run -it --name vpp --rm ligato/vpp-agent
+docker run -it --name vpp --rm ligato/vpp-agent 
+```
+   or for ARM64 platform:
```

Comment:
I think it's best to avoid putting architecture specific commands in examples rather list all supported architectures somewhere in main README. Or if you think it would be best to note here as well, you could mention it under the command section as short note that for arm64 other image should be used.

Reply:
Done.